inference/nlp/realtime/llm/opt30b/djl_deepspeed_deploy_opt30b.ipynb

```diff
@@ -0,0 +1,740 @@
+{
```

Comment:
<p>Missing a period after "lifting".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>fixed in the current push</p>